en/docs/administer/customize-the-developer-portal.md

```diff
@@ -10,30 +10,39 @@ To customize the Developer Portal theme, follow the steps given below:
 
 1. Sign in to the Choreo Console at [https://console.choreo.dev/](https://console.choreo.dev/) using a Google, GitHub, or Microsoft account.
 
-2. If you are a new user, create an organization. Enter a unique organization name. For example, "Stark Industries".
 
-    ![Create an organization in Choreo](../assets/img/references/enterprise-login/create-choreo-organization.png){.cInlineImage-small}
-
-3. In the left pane, click **Settings**.
+2. In the left pane, click **Settings**.
 
-4. In the **Organization** tab, click **Devportal Theme**.
+3. In the **API Management** tab, click **Devportal Theme**.
 
-    ![Access Devportal theme](../assets/img/administration/devportal-theme/access-devportal-theme.png)
+    ![Access Devportal theme](../assets/img/administration/devportal-theme/access-devportal-theme.png){.cInlineImage-half}
 
     Once you access the theme, you can customize the **Home** page, color theme, font, header and footer, logos, etc., by expanding the relevant sections.
 
-5. Make a change to the theme. For example, let's change the title on the **Home** page and the color theme. 
+4. Make a change to the theme. For example, let's change the title on the **Home** page and the color theme. 
 
     1. To update the title on the **Home** page, expand the **Home Page** section, and in the **Title** field, change the default text (for example, to `Try our APIs!`).
    
     2. To update the color theme, expand the **Color Palette** section, and change the colors as required (for example, change the background color to `#C3C5CD` and the primary color of the buttons to `#086634`).
    
     3. Click **Preview** to view a preview of the Developer Portal with the changes you made. Based on the changes given in the examples, the preview appears as follows.
 
-         ![Preview of customization](../assets/img/administration/devportal-theme/preview-of-customization.png)
+         ![Preview of customization](../assets/img/administration/devportal-theme/preview-of-customization.png){.cInlineImage-threeQuarters}
    
-    4. Click **Save**.
-
+    4. Click **Save** to save your changes as a draft theme.
+    
+    5. To apply the changes to the Developer Portal, click **Go Live**. To confirm that you want to go live with the changes, click **Enable** in the message that appears.
```

Comment:
Can we say **click Go Live**, since it's a switch button?

Reply:
Updated this text.